src/status_im/transport/message/v1/public_chat.cljs

```diff
@@ -30,13 +31,13 @@
    (let [{:keys [web3]} db]
      (handlers-macro/merge-fx
       cofx
-      {:db                        (assoc-in db [:transport/chats chat-id :sym-key-id] sym-key-id)
-       :shh/add-filter            {:web3       web3
-                                   :sym-key-id sym-key-id
-                                   :topic      (transport.utils/get-topic chat-id)
-                                   :chat-id    chat-id}
-       :data-store.transport/save {:chat-id chat-id
-                                   :chat    (-> (get-in db [:transport/chats chat-id])
-                                                (assoc :sym-key-id sym-key-id)
-                                                ;;TODO (yenda) remove once go implements persistence
-                                                (assoc :sym-key sym-key))}}))))
+      {:db             (assoc-in db [:transport/chats chat-id :sym-key-id] sym-key-id)
+       :shh/add-filter {:web3       web3
+                        :sym-key-id sym-key-id
+                        :topic      (transport.utils/get-topic chat-id)
+                        :chat-id    chat-id}
+       :data-store/tx  [(transport-store/save-transport-tx {:chat-id chat-id
```

Comment:
could we move the map to the next line? It's longer even than 120 chars now

Reply:
Fixing